tests/console/yast2_ftp.pm

```diff
@@ -87,19 +87,14 @@ sub run {
     script_run("rm /etc/vsftpd.conf");
     assert_script_run("rm -f /etc/vsftpd.pem");    #-f to not trigger error if file does not exist
 
+    # bsc#1183786
+    my $rsa_size = is_sle('<=12-SP2') ? '1024' : '2048';
```

Comment:
Does SLE <12SP2 simply not support RSA 2048 or do we just not want to change the test?

imho, nowadays, we should rather be sure that 2048 works as opposed to the 1024bit certs

Reply:
good point, I think is not running in SLE12 so we can remove the condition.